Add toUint, toInt and hexToUint to Strings (#5166)
OpenZeppelin/openzeppelin-contracts: .changeset/rude-cougars-look.md

---
'openzeppelin-solidity': minor
---

> `Bytes`: Add a library of common operation that operate on `bytes` objects.

Reviewer: ```suggestion
`Bytes`: Add a library for common operations on `bytes` objects.
```
